tests/common.sh

```diff
@@ -117,7 +117,10 @@ function pod_count () {
 
 function apply_operator {
   operator_requirements
-  kubectl apply -f resources/operator.yaml
+  BENCHMARK_OPERATOR_IMAGE=${BENCHMARK_OPERATOR_IMAGE:-"quay.io/benchmark-operator/benchmark-operator:master"}
```

Comment:
This seems redundant with the update operator image function below where it patches together the image based on environment variables.

From top of common.sh
```
image_location=${RIPSAW_CI_IMAGE_LOCATION:-quay.io}
image_account=${RIPSAW_CI_IMAGE_ACCOUNT:-rht_perf_ci}
```